src/components/player/index.jsx

```
@@ -9,8 +9,9 @@ import {
   likeUnlikeVideo,
 } from "../../app/features/videomania";
 import AppInput from "../form/AppInput";
+import ProfileCard from "../card/ProfileCard";
 
-const Player = ({ video, isOpen, onClose }) => {
+const Player = ({ video, isOpen, onClose, isTop = false }) => {
```

ESLint

[error] 'video' is missing in props validation

(react/prop-types)